book/chapters/config.py

```diff
@@ -84,6 +85,13 @@ def deliver_files(query, s: sample):
     assert files is not None, "No files returned from deliver! Internal error"
     return files
 
+def flat_array_from_files(files, branch_name):
+    data = []
+    for file_path in files:
+        file = uproot.open(file_path)
+        array = file['atlas_xaod_tree;1'][branch_name].arrays(library="ak")[branch_name]
```

Comment:
Don't use the `;1` - that is implied and you always want to use the latest version. (you can see `;2` in there - this is an old notation left over from an operating system called VMS.